server/src/app.ts

```
@@ -6,7 +6,7 @@
 import { rateLimit } from 'express-rate-limit';
 import cors from 'cors';
 import userRoute from './routes/users.route';
-
+import path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)